src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -166,6 +171,27 @@ class CardsResource extends Resource {
         created(cardResult).build()
     }
 
+    @PUT
+    @Path('{id}')
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
I think you should be able to remove this from the `PUT` and `POST` methods and just add it to the class level, since the only thing this resource consumes is JSON.